kobotoolbox/kpi
# feat(organizations): update is_mmo to identify mmo subscriptions TASK-1231

### 🗒️ Checklist

1. [x] run linter locally
2. [x] update all related docs (API, README, inline, etc.), if any
3. [x] draft PR with a title `<type>(<scope>)<!>: <title> TASK-1234`
4. [x] tag PR: at least `frontend` or `backend` unless it's global
5. [x] fill in the template below and delete template comments
6. [x] review thyself: read the diff and repro the preview as written
7. [x] open PR & confirm that CI passes
8. [x] request reviewers, if needed
9. [ ] delete this section before merging

### 📣 Summary
Change the logic of the `is_mmo` property on the Organization model to check Stripe product metadata for the `'mmo_enabled'` field.


### 👀 Preview steps
This change is covered by a unit test.

For a manual test:
1. Create an organization and purchase a storage addon (this is a subscription in Stripe even if we don't describe it as such in the UI).
2. Check org's `is_mmo` property in shell. On `main`, this will be `True` because it is only checking to see whether or not an org has a plan. On this branch, however, it will correctly be `False` because storage addons shouldn't enable MMO features.
3. Upgrade to an enterprise plan, which includes MMO support. 
4. Check `is_mmo` property in shell. On this branch, it should now be `True`.
